src/utils/getMentions.js

```diff
@@ -0,0 +1,22 @@
+import iterateMentionsMarkup from './iterateMentionsMarkup'
+
+const getMentions = (value, config) => {
+  const mentions = []
+  iterateMentionsMarkup(
+    value,
+    config,
+    () => {},
```

Comment:
The second time I see this no-op being passed here. Does it make sense to make this param the last one and then turn it optional?